fixtures/browser.py

```diff
@@ -106,6 +107,11 @@ def browser():
     return utils.browser.browser
 
 
+@pytest.fixture(scope='function')
+def create_view():
+    return lambda *args, **kwargs: current_appliance.browser.instantiate_view(*args, **kwargs)
```

Comment:
i propose using utils.appliance.get_or_create_appliance here to make the point of the usage more clear